detailed_docs/RC/OnRCStatus.md

```diff
@@ -0,0 +1,148 @@
+## Use Case 1: Notification about RC modules allocation during application’s registration
+
+**Main Flow:**
+
+_Pre-conditions:_  
+
+a.  Mobile device is connected to SDL  
+
+b.	RC functionality is allowed on HMI  
+
+c.	All modules are free  
+
+d.	RC functionality is allowed by policies  
+
+_Steps:_
+
+1.	RC application starts registration (upon a OnButtonPress or SetInteriorVehicleData request)
+
+_Expected:_   
+1. SDL sends OnRCStatus with all modules in `freeModules` parameter to HMI
+2. SDL sends OnRCStatus(allowed = true, freeModules = {}) notification and to registered mobile application
+
+**Alternative flow 1**  
+b.1 RC functionality is disallowed on HMI  
+b.2 SDL sends OnRCStatus(allowed = false, freeModules = {}, allocatedModules = {}) notification to registered mobile application 
+
+**Alternative flow 2**  
+1.a.1 Non-RC application starts registration  
+1.a.2 SDL does not sends OnRCStatus to HMI and non-RC application
+
+
+## Use Case 2: Notification about RC module allocation after application’s registration
+
+**Main Flow:**
+
+_Pre-conditions:_  
+
+a.	Mobile device is connected to SDL  
+
+b.	RC functionality is allowed on HMI  
+
+c.	All modules are free  
+
+d.	RC functionality is allowed by policies  
+
+e.	RC app_1 is registered 
+
+_Steps:_
+1.	RC app_1 sends control request to module_1  
```

Comment:
`RC app_1 sends control request to module_1`  (upon a OnButtonPress or SetInteriorVehicleData request)?

Reply:
@yang1070, please see the updates in https://github.com/smartdevicelink/sdl_requirements/pull/89/commits/2e9692b47878e6f1b766e7fbd380673c29af74db